src/dpo/src/Optdp.cpp

```diff
@@ -729,7 +740,19 @@ void Optdp::createArchitecture()
     unsigned orient = dbToDpoOrient(row->getOrient());
     archRow->setOrient(orient);
   }
-
+  for (auto skip : skip_list) {
+    std::string skip_string = "[";
+    for (int i = 0; i < skip.second.size(); i++) {
+      skip_string += skip.second[i] + ",]"[i == skip.second.size() - 1];
+    }
+    logger_->warn(DPO,
+                  108,
+                  "Skipping the rows with height {} as the minimum row height "
+                  "is {}. Here is the skip list: {}",
```

Comment:
A large design may have many rows.  I'm concerned this may produce a huge message string.  I think it might be better to say "skipping all rows with the following SITE types ..." rather than list all the rows themselves.  The skip_list could just be a set of sites.